modules/monitoring-configuring-a-local-persistent-volume-claim.adoc

```diff
@@ -40,12 +38,12 @@ data:
           storageClassName: *_storage class_*
           resources:
             requests:
-              storage: *_40Gi_*
+              storage: *_amount of storage_*
```

Comment:
metadata:
      name: *_PVC name_*
actually it is not PVC name, it is PVC name prefix
eg:
 if name: prometheus for prometheusK8s
the pvc would be
prometheus-prometheus-k8s-0
prometheus-prometheus-k8s-1

Reply:
Implemented in #16978.